packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -169,7 +172,11 @@ public Bitmap buildMarker(String type, String text) {
             case "price":
                 return getPriceMarkerBitmap(text);
             case "rounded":
-                return getRoundedMarkerBitmap(text);
+                return getRoundedMarkerBitmap(text, Color.WHITE, Color.BLACK);
```

Comment:
Since the visited and selected markers are the same shape i added two parameters to change the text and marker color